cmd/analyzer/parse_args.go

```diff
@@ -210,12 +218,9 @@ func errorInErgs(args *InArgs, flagset *flag.FlagSet) error {
 	return nil
 }
 
+//gocyclo:ignore
 func notSupportedYetArgs(args *InArgs) error {
 	diffAnalysis := *args.AnalysisType == allEndpointsDiff || *args.AnalysisType == allSubnetsDiff
-	if !diffAnalysis && *args.AnalysisType != allEndpoints && *args.OutputFormat != TEXTFormat &&
-		*args.OutputFormat != JSONFormat {
-		return fmt.Errorf("currently only txt/json output format supported with %s analysis type", *args.AnalysisType)
-	}
 	if diffAnalysis && *args.OutputFormat != TEXTFormat && *args.OutputFormat != MDFormat {
 		return fmt.Errorf("currently only txt/md output format supported with %s analysis type", *args.AnalysisType)
```

Comment:
this `if` (`if diffAnalysis && *args.OutputFormat != TEXTFormat && *args.OutputFormat != MDFormat`) can also be deleted, is covered by the code you added that checks the table of supported formats per analysis type.

Reply:
done